### ISSUE TITLE
Database Schema

### ISSUE THREAD

**Issue by AdamYLiang** [JMBourgoin/Everwrote#2]
**All Tables**
- [x] Add `updated_at` column since timestamps gives it to us

**Users Table**
- [x] `session_token` should have unique and indexed in details section 

**Notebooks Table**
- [x] `author_id` should not be unique (collaborators will also be able to access the notebook)
- [x] `author_id` should be a foreign key

**Notes Table**
- [x] Should have a body column

**Tags**
- [x] A joins table between tags and notes will make the associations more clear
Basically this joins table will represent a many-to-many relationship between tags and notes. Notes will go through the joins table to get all of it's tags. 
Tags should also be uniquely scoped on notes so that you can't give a note the same tag multiple times.

Let me know if you have any questions or want to talk about any of these!

**Comment by JMBourgoin** [comment]
For Notebooks Table:
Isn't author_id listed as a foreign key already?  Are you just wanting me to note that in the details column?

For Tags joins table:  
I've created another table -- is this in the right direction?

**Comment by AdamYLiang** [comment]
Nice changes so far and the joins table looks good.
One more tweak:
- [ ] Add `foreign key` for details on all foreign keys in tables

Let me know if there are any questions or if you want to talk about the schema!
